### PR TITLE
Update documentation with logo - prepare for go-live

### PR DESCRIPTION
**Pull request by DovOps**
THIS SOFTWARE IS CONTRIBUTED SUBJECT TO THE TERMS OF THE FINOS Corporate Contributor License Agreement.

THIS SOFTWARE IS LICENSED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE AND ANY WARRANTY OF NON-INFRINGEMENT, ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. THIS SOFTWARE MAY BE REDISTRIBUTED TO OTHERS ONLY BY EFFECTIVELY USING THIS OR ANOTHER EQUIVALENT DISCLAIMER IN ADDITION TO ANY OTHER REQUIRED LICENSE TERMS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,51 +2,60 @@
...
 
-## Current Project Status
 
-This is currently a Work-In-Progress. At the moment there are some components which are just placeholder API contracts for an implementation to be created, while others are already runnable reference implementations.
+## Project Demo and Overview Presentation
+
+Learn more about the project - including a brief demo, in the Keynote Demo session 
+that was presented at the [Open Source in Finance Forum 2023](https://events.linuxfoundation.org/open-source-finance-forum-new-york/)
+
+[![FINOS - Incubating](./docs/img/2023_osff_video_thumb.png)](https://youtu.be/fO-T3V-GKBs?t=326)
```
Comment (TheJuanAndOnly99) [inline README.md:31]: Small change but I would change this to say `TraderX - Demo` instead of `FINOS - Incubating`

File: README.md [hunk 2]
```diff
@@ -2,51 +2,60 @@
...
-This section will be filled out once the code is in place.
+## Installation  
 
-## Usage example (WIP)
+This is installed locally through normal git clone operations.
 
-In order to get things working together, it is recommended to select a range of ports to provde all running processes with, so that the pieces can interconnect as needed.  A more advanced instance of this project would do things using container/service location abstractions.  Here's one such example convention.
+## Usage example (Simple)
+
+In order to get things working together, it is recommended to select a range of ports to provde all running processes with, so that the pieces can interconnect as needed.  A more advanced instance of this project is possible using the docker compose files which are present in each project directory. To run this all up 'by hand' here are default ports which are used, and you can easily export these variables to your favorite shell.
```
Comment (TheJuanAndOnly99) [inline README.md:58]: provide*

Comment (DovOps) [reply README.md]: Thanks. Fixed

File: README.md (hunk 1 above)
Comment (DovOps) [reply README.md]: Thanks. Fixed

**Pull request merged**